### ISSUE TITLE
[Bug]: Route Autocompletion Doesn't Include Changes In RouteServiceProvider

### ISSUE THREAD

**Issue by ZacharyDuBois** [laravel-idea/plugin#412]
### Bug description

(First off, love this plugin!)

It seems if you set a name on the route group for the API endpoints (in my case, I prefixed all the names with `api.`) in the `RouteServiceProvider`, it will not autocomplete with that prefix
<img width="366" alt="Screen Shot 2021-09-29 at 15 09 41" src="https://user-images.githubusercontent.com/2458289/135333197-c1531634-65d5-44dc-adf3-493ff4e1bc4c.png">
.

### Plugin version

4.4.0.212

### Operating system

MacOS

### Steps to reproduce

`app/Providers/RouteServiceProvider.php` in `boot()`

```php
$this->routes(function () {
    Route::prefix('api')
        ->middleware('api')
        ->name('api.')
        ->namespace($this->namespace)
        ->group(base_path('routes/api.php'));

    Route::middleware('web')
        ->namespace($this->namespace)
        ->group(base_path('routes/web.php'));
});
```

`routes/api.php`

```php
Route::prefix('/special')->name('special.')->group(function () {
    Route::post('/endpoint', SpecialEndpointController::class)
         ->name('endpoint');
});
```

Attempt to autocomplete the doing something like `route('api.special.endpoint')`. It shows only `special.endpoint` and not `api.special.endpoint`.

**Comment by adelf** [comment]
Hi.

`->namespace($this->namespace)` whatis the value of the `namespace` field?
If it's null or doesn't exist - this is not working...

**Comment by ZacharyDuBois** [comment]
<img width="219" alt="Screen Shot 2021-09-29 at 15 15 44" src="https://user-images.githubusercontent.com/2458289/135334105-11a7246d-1561-4578-b217-1dd5b5120bcb.png">

So this is a vanilla JetStream installation. `$this->namespace` is indeed `null`. I will check with a vanilla Laravel 8.x app and see if it is the same.

**Comment by adelf** [comment]
This happens with a lot of installations... It seems I need to fix it somehow... even for namespace = null...

**Comment by ZacharyDuBois** [comment]
<img width="514" alt="Screen Shot 2021-09-29 at 15 23 40" src="https://user-images.githubusercontent.com/2458289/135335048-2eac3aa2-c5fd-4de8-8d1a-835314011ab6.png">
Yep, this is from a fresh Laravel `v8.6.2` installation created with `$ laravel new Testing`. Hope this helps.

**Comment by adelf** [comment]
@ZacharyDuBois you can just remove `->namespace($this->namespace)` and it should work. I'll try to fix it also from my side...

**Comment by ZacharyDuBois** [comment]
<img width="438" alt="Screen Shot 2021-09-29 at 15 29 58" src="https://user-images.githubusercontent.com/2458289/135335837-50c5422d-38fa-426d-bb1a-9441f6709c25.png">

Worked, awesome, thanks!

**Comment by adelf** [comment]
It should be fixed by the 4.4.2 version.
